exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) {
+    return "A";
+  } else if (mark < 80 && mark > 60) {
```

Comment:
JavaScript checks if/else conditions starting from the top and picks the first one that is "true". 

We already know that mark is **not** >= 80 (because if it is >= 80 we would return "A" and would not check other `else` branches). The question is - should we check that `mark < 80`? Can we simplify this condition (and the following ones)?